### PR TITLE
[#129] Create a new adapter called PostgreSQLLobeDB

### LINKED ISSUE #129: Create a new adapter called RedisPostgresqlDB
Parent Card: https://github.com/orgs/singnet/projects/6?pane=issue&itemId=57550293

### PR DESCRIPTION
**Pull request by marcocapozzoli**
Resolves #129

### CODE REVIEW

File: hyperon_das_atomdb/exceptions.py [hunk 1]
```diff
@@ -44,3 +44,7 @@ class RetryException(BaseException):
 
 class InvalidAtomDB(BaseException):
     ...  # pragma no cover
+
+
+class ParserException(BaseException):
```
Comment (andre-senna) [inline hyperon_das_atomdb/exceptions.py:49]: This name is too generic. Firstly. I suggest InvalidSQL. We're not claiming to actually parse SQL. To do that we would need to be prepared for anything that's possible in SQL and this definitely is not the case. We want to be able to not implement some specific SQL feature which is particularly difficult to map to our Atomspace.

File: tests/integration/adapters/test_redis_postgresqllobe_db.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                """
+                CREATE TABLE IF NOT EXISTS users (
+                    ID SERIAL PRIMARY KEY,
+                    name VARCHAR(100),
+                    age INTEGER
+                );
+                """
+            )
+            cursor.execute("INSERT INTO users (name, age) VALUES ('Adam', 30);")
+            cursor.execute("INSERT INTO users (name, age) VALUES ('Eve', 31);")
```
Comment (andre-senna) [inline tests/integration/adapters/test_redis_postgresqllobe_db.py:34]: Shouldn't Eve be younger than Adam? ;-)

**Review by andre-senna — APPROVED** [review]
Kudos! :clap: Masterfully done!

File: tests/integration/adapters/test_redis_postgresqllobe_db.py (hunk 2 above)
Comment (marcocapozzoli) [reply tests/integration/adapters/test_redis_postgresqllobe_db.py:34]: :smile:
Yeah. But women must come first, we're nothing without them :smiley:

File: hyperon_das_atomdb/exceptions.py (hunk 1 above)
Comment (marcocapozzoli) [reply hyperon_das_atomdb/exceptions.py]: Done!

**Pull request merged**